filesystem/fat32/fat32.go

```diff
@@ -637,18 +637,130 @@ func (fs *FileSystem) OpenFile(p string, flag int) (filesystem.File, error) {
 	}, nil
 }
 
-// Rename renames (moves) oldpath to newpath. If newpath already exists and is not a directory, Rename replaces it.
-//
-//nolint:revive // parameters will be used eventually
-func (fs *FileSystem) Rename(oldpath, newpath string) error {
-	return filesystem.ErrNotImplemented
-}
-
 // removes the named file or (empty) directory.
 //
 //nolint:revive // parameters will be used eventually
```

Comment:
As the method got implemented, lint exception is not needed anymore here. Please remove it.